pkg/gcc/acknowledgment.go

```diff
@@ -0,0 +1,26 @@
+package gcc
```

Comment:
Do you think we could move this to `internal/cc`

This seems like something we could share between congestion controllers. Would love to see this directory be dedicated to gcc specific stuff!

Reply:
Yes, I think we could move it out of this package, but I'm not sure what the right place would be, `internal/cc` would mean that `pkg/gcc` can basically only be used from this module, right? I think it would be nice to allow using `pkg/gcc` without interceptors?

Reply:
Yes, I think that would work, but currently I use a default `BandwidthEstimator` in the interceptor, so that would create an import cycle. But maybe there's a better way to set the default estimator without including the package just for that?

Reply:
*and the default uses `gcc.NewSendSideBWE()`

Reply:
We could maybe do `twcc` for now and fix in the future?